tests/run-tests.sh

```diff
@@ -19,9 +19,9 @@ CENTOS_MVAPICH2_PATH="/opt/mvapich2-2.3.5"
 CENTOS_MVAPICH2X_PATH="${MVAPICH2X_INSTALLATION_DIRECTORY}/gnu9.2.0/mofed5.1/azure-xpmem/mpirun"
 CENTOS_OPENMPI_PATH="/opt/openmpi-4.0.5"
 
-UBUNTU_MOFED_VERSION="MLNX_OFED_LINUX-5.2-1.0.4.0"
+UBUNTU_MOFED_VERSION="MLNX_OFED_LINUX-5.2-2.2.0.0"
```

Comment:
Wouldn't this break 20.04? Ideally, we should update MOFED/HPC-X on both 18.04 and 20.04.